x/ecosystem-incentive/keeper/incentive_unit_service.go

```diff
@@ -67,9 +92,30 @@ func (k Keeper) GetIncentiveUnit(ctx sdk.Context, id string) (types.IncentiveUni
 	return incentiveUnit, true
 }
 
+func (k Keeper) GetIncentiveUnitIdsByAddr(ctx sdk.Context, address sdk.AccAddress) types.IncentiveUnitIdsByAddr {
+	store := ctx.KVStore(k.storeKey)
+	prefixStore := prefix.NewStore(store, []byte(types.KeyPrefixIncentiveUnitIdsByAddr))
+
+	bz := prefixStore.Get(address)
+	if len(bz) == 0 {
```

Comment:
I think `if bz == nil` would be the correct case.

Reply:
I modified.